test/audits/organic-keywords.test.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* eslint-env mocha */
+
+import { createSite } from '@adobe/spacecat-shared-data-access/src/models/site.js';
+
+import chai from 'chai';
+import sinonChai from 'sinon-chai';
+import chaiAsPromised from 'chai-as-promised';
+import sinon from 'sinon';
+import nock from 'nock';
+import auditOrganicKeywords from '../../src/organic-keywords/handler.js';
+
+chai.use(sinonChai);
+chai.use(chaiAsPromised);
+const { expect } = chai;
+
+describe('Organic Keywords Tests', () => {
+  let message;
+  let context;
+  let dataAccessMock;
+
+  const sandbox = sinon.createSandbox();
+
+  const siteData = {
+    id: 'foo',
+    baseURL: 'https://foobar.com',
+  };
+  const site = createSite(siteData);
+
+  const auditResult = {
+    keywords: [
+      {
+        keyword: 'foo',
+        sum_traffic: 123,
+        best_position: 1,
+        best_position_prev: 1,
+        best_position_diff: 0,
```

Comment:
This is something we could easily compute ourselves. How much does it cost to get this field for every keyword every time we run the audit for a site?

Reply:
it's just 1 unit, so I thought it might save us some performance. Is it worth saving one unit?